geopandas/io/arrow.py

```diff
@@ -606,6 +643,19 @@ def _read_parquet(path, columns=None, storage_options=None, **kwargs):
         both ``pyarrow.fs`` and ``fsspec`` (e.g. "s3://") then the ``pyarrow.fs``
         filesystem is preferred. Provide the instantiated fsspec filesystem using
         the ``filesystem`` keyword if you wish to use its implementation.
+    bbox: tuple, optional
+        Bounding box to be used to filter selection from geoparquet data. This
+        is only usable if the data was saved with the bbox covering metadata.
+        Input is of the tuple format (xmin, ymin, xmax, ymax).
+    read_bbox_column: bool, default False
+        The bbox column is a struct with the minimum rectangular box that
+        encompasses the geometry. It is computationally expensive to read
+        in a struct into a GeoDataFrame. As such, it is default to not
+        read in this column unless explictly specified as True.
+        If  ``columns`` arguement is used and contains ``bbox``,
+        this will override ``read_bbox_column`` and include ``bbox``
+        even if this is False.
```

Comment:
(line wrapping in this suggestion is likely not right)
```suggestion
        a struct into a GeoDataFrame. As such, the default is to
        skip reading this columns.
        If  the bbox column name specified in parquet metadata (under the the "covering" key) 
        is specified in ``columns``, then the column will always be included and the value of 
        ``read_bbox_column`` is ignored.
```